samples/fan_out_fan_in_tensorflow/local.settings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "FUNCTIONS_WORKER_RUNTIME": "python",
+    "AzureWebJobsStorage": "<your connection string>",
```

Comment:
Looks like we're using a placeholder here instead of the empty string which I saw in other samples. I suggest we try to be consistent across the samples so that our users know what to expect. I'll let you decide which of the two options (empty string vs. placeholder) is preferred. :)